### ISSUE TITLE
glusterfs: /mnt/container-volumes not mounted on boot

### ISSUE THREAD

**Issue by ryane** [CiscoCloud/microservices-infrastructure#872]
pre-reboot:

```
$ sudo mount | grep gluster
fs.glusterfs.service.consul:/container-volumes on /mnt/container-volumes type fuse.glusterfs (rw,relatime,user_id=0,group_id=0,default_permissions,allow_other,max_read=131072)
```

post-reboot:

```
$ sudo mount | grep gluster
$
```

**Comment by ryane** [comment]
this issue is still outstanding

**Comment by AndriiOmelianenko** [comment]
Hi,

Does it related to the fact that created vdb volumes aren't getting mounted at all (even without reboot)? :
```
[cloud-user@ta-control-01 ~]$ mount | grep vdb
[cloud-user@ta-control-01 ~]$ df -h
Filesystem      Size  Used Avail Use% Mounted on
/dev/vda1        47G  1.7G   44G   4% /
devtmpfs        7.9G     0  7.9G   0% /dev
tmpfs           7.9G     0  7.9G   0% /dev/shm
tmpfs           7.9G   33M  7.9G   1% /run
tmpfs           7.9G     0  7.9G   0% /sys/fs/cgroup
tmpfs           1.6G     0  1.6G   0% /run/user/0
tmpfs           1.6G     0  1.6G   0% /run/user/1000

[cloud-user@ta-control-01 ~]$ ll /dev/vd*
brw-rw----. 1 root disk 253,  0 Dec 18 11:47 /dev/vda
brw-rw----. 1 root disk 253,  1 Dec 18 11:47 /dev/vda1
brw-rw----. 1 root disk 253, 16 Dec 18 12:11 /dev/vdb
```
cluster provisioned with last master code

**Comment by BrianHicks** [comment]
@avnik does that look related to you?

**Comment by avnik** [comment]
On Fri, Dec 18, 2015 at 06:28:49AM -0800, Brian Hicks wrote:

> @avnik does that look related to you?
> 

I'll take look now.

**Comment by avnik** [comment]
On Fri, Dec 18, 2015 at 06:28:49AM -0800, Brian Hicks wrote:

> @avnik does that look related to you?
> 

Looks like another tied-in problem.

**Comment by avnik** [comment]
@AndriiOmelianenko  /dev/vdb going to be LVM physical volume (PV), and used for /dev/mantl/gluster and for docker thin provision)

**Comment by larry-svds** [comment]
I just made a pull request for a fix.  There is a discussion if you want to solve this on your systems before a merge.  There might be a better way as my pull request required a modified base unit file.

https://github.com/CiscoCloud/microservices-infrastructure/pull/952

**Comment by ryane** [comment]
fixed by #928
